components/Block.vue

```diff
@@ -229,6 +219,7 @@ export default {
 <style>
 .Block {
   @apply relative;
+  user-select: none;
```

Comment:
this is unrelated but wanted to put it in to remove some funky selection behavior